src/ng-fab-form-directive-f.js

```diff
@@ -130,6 +130,16 @@ angular.module('ngFabForm')
                                         inputElCtrl.$render();
                                     }
                                 }
+
```

Comment:
We could also use `element[0].querySelectorAll('input, select');` to not have duplicate logic there.